### DISCUSSION TITLE
2.0.1 in Ubuntu

### DISCUSSION THREAD

**Discussion by Codyport** [sezanzeb/input-remapper#817] [Q&A]
Hi all, 2.0.0 is the latest version showing in Discover on Kubuntu 23.10 (and I assume therefore Ubuntu).  Trying to install 2.0.1 from the downloaded .deb results in:

Failed to enable unit: Unit file input-remapper.service does not exist.
Failed to start input-remapper.service: Unit input-remapper.service not found.

.. which at this point has me stumped.   I'm trying to upgrade from 2.0.0 -> 2.0.1 in the hope it will resolve an issue I'm having whereby periodically the mappings stop working, and I have to start the GUI and re-apply them.

**Comment by Mindus-Amitiel-Debsin** [comment]
Hello.
I'm just a regular user, but I had some issues with installing it too (a different error) and I solved it and got it working using these steps. I'm on Pop!_OS 22.04, a derivative of Ubuntu.

sudo apt update
sudo apt purge input-remapper -y
sudo apt update
sudo apt autoremove -y
sudo apt clean

I am not an expert, but I think this will allow you to install it again without a problem. You might want to backup your remapped keys manually before you uninstall it.
I hope I helped!

**Reply by Codyport** [reply]
Hi!  Thanks... getting rid of all the services and places that Input Remapper drops things was a challenge indeed.  I eventually moved to 1.5.1 and all my issues were solved.  With v2, my mouse mappings would frequently just stop working - with v1.5, everything is fine.  I had to use "apt-mark hold" on the package as it was absolutely determined to upgrade back to V2.

PS purging v2 still left detritus all over the place which broke v1.5.  Probably because I'd tried so many methods of making v2 not quit working.
